yarn.lock

```diff
@@ -1602,6 +1602,51 @@
     zen-observable "^0.8.15"
     zod "^3.11.6"
 
+"@backstage/core-components@^0.9.6-next.1":
```

Comment:
These additions are here because of outdated dependency ranges in `package.json`, need to make sure all deps are referring to the current version in the repo